gas/testsuite/gas/riscv/zc-zca-cm-lh-fail-march.s

```diff
@@ -0,0 +1,9 @@
+zcb_lh:
+	# test to compress lh
```

Comment:
nit: comments like this don't add much, from the file name and the symbol name that's implied.

Reply:
Fixed in next patch.